python/src/scipp/plot/plot.py

```diff
@@ -5,18 +5,19 @@
 # Scipp imports
 from .._scipp import core as sc
 from .. import _utils as su
-from .sciplot import SciPlot
 from .tools import make_fake_coord
 
 
 def plot(scipp_obj,
+         interactive=True,
          projection=None,
          axes=None,
          color=None,
          marker=None,
          linestyle=None,
          linewidth=None,
          bins=None,
+         is_doc_build=False,
```

Comment:
This is an odd option to have on a public function that every user of scipp will use. Can we avoid this?

Reply:
note that this is not fully public, since we now have a wrapper `plot` function in the `__init__` which imports `plot` and `_plot`.

Reply:
I found a better way of doing this. See https://github.com/scipp/scipp/pull/1324/commits/81ad572389be3929f1edf215ba1a72d4611bf020